espanol/canudos.md

```diff
@@ -115,3 +115,5 @@ Lo que nadie sabía, es que el consejero era un versado maestro instruído en la
 No tuvo más opción que buscar refuerzos, luchadores incanzables que tuvieran las siguientes características:
 
 - [Características](caracteristicas/caracteristicas.md)
+
+Porque en el fondo él sabía que por mucho que hubiese ganado esta batalla, la guerra no había hecho más que [comenzar](guerra/duracion.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.